applications/irda/irda-app-file-parser.cpp

```diff
@@ -306,13 +399,15 @@ bool IrdaAppFileParser::check_errors() {
 }
 
 std::string IrdaAppFileParser::file_select(const char* selected) {
-    TextStore* filename_ts = new TextStore(128);
+    TextStore* filename_ts = new TextStore(IrdaAppRemoteManager::max_remote_name_length);
```

Comment:
Probably it's better to use std::unique_ptr there. There is no reason to use a raw C pointer. Also, this simplifies the mamory management here( delete will be called automatically)